server/core/src/main/java/dev/slimevr/tracking/trackers/udp/TrackersUDPServer.kt

```diff
@@ -391,6 +393,17 @@ class TrackersUDPServer(private val port: Int, name: String, private val tracker
 					"[TrackerServer] User action from ${connection.descriptiveName } received. $name reset performed."
 				)
 			}
+
+			is UDPPacket22FeatureFlags -> {
+				if (connection == null) return
+				// Respond with server flags
+				bb.limit(bb.capacity())
+				bb.rewind()
+				parser.write(bb, connection, packet)
```

Comment:
Does it just send the packet back without actually setting anything that server supports?

Reply:
It's abstracted into `writeData` in packet classes:
https://github.com/SlimeVR/SlimeVR-Server/blob/f84efc413dc7cb0e4b810609a17aaab55491c92f/server/core/src/main/java/dev/slimevr/tracking/trackers/udp/UDPProtocolParser.kt#L34-L38

Reply:
But the packed is the same that was received from the tracker.

Reply:
It writes a fixed value, different from what server had received.

```kotlin
: UDPPacket(22) {
	override fun readData(buf: ByteBuffer) {
		firmwareFeatures = FirmwareFeatures.from(buf, buf.remaining())
	}

	override fun writeData(buf: ByteBuffer) {
		buf.put(ServerFeatureFlags.packed)
	}
```

Do we want server flags to be toggleable from the UI? If so, we can make `ServerFeatureFlags` writable, but the firmware only accepts 1 flags packet per connection, will need to remove that check.